packages/react-components/components/TextButton.v2.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * A button that's just text. For ann underlined text link, see Link.tsx
+ */
+
+import Touchable, { Props as TouchableProps } from '@celo/react-components/components/Touchable'
+import colors from '@celo/react-components/styles/colors.v2'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import * as React from 'react'
+import { StyleProp, StyleSheet, Text, TextStyle } from 'react-native'
+
+type Props = TouchableProps & {
+  style?: StyleProp<TextStyle>
+  testID?: string
+}
+// unstyled Touchable Text, good for making other Text Buttons such as TopBarButton
+export function TextButtonGeneric(props: Props) {
```

Comment:
nit: Maybe this could be named `BorderlessButton`? and moved to its own file since it's used elsewhere.